src/marqo/s2_inference/model_registry.py

```diff
@@ -2008,7 +2058,9 @@ def _get_model_load_mappings() -> Dict:
             "fp16_clip": FP16_CLIP,
             'random':Random,
             'hf':HF_MODEL,
-            "no_model": NO_MODEL}
+            "no_model": NO_MODEL,
+            #"languagebind": LANGUAGEBIND,
```

Comment:
why is this commented out?

Reply:
Because currently we use different loading logic for Multimodal models https://github.com/marqo-ai/marqo/blob/450fea3adc74929f3c64a18375ca17b9e0ef2afd/src/marqo/s2_inference/s2_inference.py#L551